packages/clay-css/src/scss/variables/_forms.scss

```diff
@@ -697,6 +697,8 @@ $input-group-inset-item-btn-monospaced: map-deep-merge(
 	(
 		font-size: 1rem,
 		height: 2rem,
+		margin-bottom: math-sign($input-border-bottom-width),
+		margin-top: math-sign($input-border-top-width),
```

Comment:
@pat270 this will only affect the `input-group-inset` with `btn-monospaced` right?

Reply:
@matuzalemsteles Yes, you can see at https://github.com/liferay/clay/blob/1c3f5839afd8a9881dfc4906ac204c27de887e78/packages/clay-css/src/scss/components/_input-groups.scss#L529

Reply:
Thank you @pat270!